src/components/NewComment.jsx

```diff
@@ -0,0 +1,42 @@
+import { postComment } from "../axiosFunctions";
+import { useState } from "react";
+const NewComment = ({ article_id, comments, setComments }) => {
+  const [input, setInput] = useState("");
+  const inputHandler = (e) => {
+    setInput(e.target.value);
+  };
+  const submitHandler = (e) => {
+    e.preventDefault();
+    postComment(article_id, { username: "cooljmessy", body: input }).then(
```

Comment:
fine to hardcode this for now - if you get to implementing useContext you could replace it later